dev/integration_tests/deferred_components_test/run_release_test.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Copyright 2014 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+rm -f build/app/outputs/bundle/release/app-release.apks
+rm -f build/app/outputs/bundle/release/run_logcat.log
+
+flutter build appbundle
+
+java -jar $1 build-apks --bundle=build/app/outputs/bundle/release/app-release.aab --output=build/app/outputs/bundle/release/app-release.apks --local-testing
+java -jar $1 install-apks --apks=build/app/outputs/bundle/release/app-release.apks
+
+adb shell "
+am start -n io.flutter.integration.deferred_components_test/.MainActivity
+sleep 12
+exit
+"
+adb logcat -d -s "flutter" > build/app/outputs/bundle/release/run_logcat.log
```

Comment:
this works if we are using android emulators with `-wipe-data` or if there's nothing cached between CI runs. Do you know if this would be the case?

e.g. consider the case where job 1 runs, then job 2 runs and sees the same log line from job 1.

Reply:
I'll go find out :)

Reply:
Added filters/timestamping to ensure only the latest run's logs are tracked.